test/controllers/payments/WaysToPayControllerSpec.scala

```diff
@@ -155,12 +157,17 @@ class WaysToPayControllerSpec extends AmlsSpec with AmlsReferenceNumberGenerator
               "waysToPay" -> WaysToPay.Card.entryName
             )
 
+//            when {
+//              controller.paymentsService.requestPaymentsUrl(any(), any(), any(), any(), any())(any(), any(), any())
+//            } thenReturn Future.successful(NextUrl("/payments-next-url"))
+
```

Comment:
Can we please remove this if not needed? Should this test be there and should it pass? I.e. did it pass before the auth changes?

Reply:
Removed